src/DumpPathsAsMsgPackDev.h

```diff
@@ -33,6 +33,25 @@ class Mat2x3 {
   }
 };
 
+class Path {
```

Comment:
I would call the class `Point` rather than `Path`. Path to me implies many points.